src/main/resources/CodyBundle.properties

```diff
@@ -157,6 +157,7 @@ filter.sidebar-panel-ignored-file.learn-more-cta=Learn more
 # Other Actions
 action.cody.restartAgent.text=Restart Cody Agent
 chat.enhanced_context.title=Chat Context Settings
+action.sourcegraph.disabled.description=Log in to Sourcegraph to enable Cody features
```

Comment:
Can you please put it in plugin.xml where all others are?

Reply:
Sadly, IntelliJ SDK does not support text for disabled actions. Maybe we should not modify the descriptions at all? Anyway, if we want to keep the desc it must remain as it is.